test/regression/2016-04-14-sdiv-2.c_

```diff
@@ -0,0 +1,25 @@
+// XFAIL: *
```

Comment:
Thanks, that's just a way to add a currently disabled test case.  Is there a better way to do that?